airflow-core/src/airflow/ui/public/i18n/locales/he/dashboard.json

```diff
@@ -8,7 +8,7 @@
   },
   "group": "קבוצה",
   "health": {
-    "dagProcessor": "Dag מנתח",
+    "dagProcessor": "מנתח Dags",
```

Comment:
Otherwise it looks flipped

Reply:
This needs to be fixed by applying/removing the RTL label from the text field correctly